boot/freeldr/bootsect/fat32.S

```diff
@@ -260,6 +260,7 @@ ReadSectorsCHSLoop:
 
     loop ReadSectorsCHSLoop                     // Read next sector
 
+    pop es
```

Comment:
Is this due to a bug in these old BIOSes, or a bug in our code where we pushed ES, but forgot to pop ES ?

Reply:
Yes. Don't know why the test of this failed 2 weeks ago and I tried to find a further issue. But that's all!